libs/cms/src/lib/loaders/organizationLogo.loader.ts

```diff
@@ -3,27 +3,40 @@ import { Injectable } from '@nestjs/common'
 
 import { NestDataLoader } from '@island.is/nest/dataloader'
 import { CmsContentfulService } from '../cms.contentful.service'
+import { IOrganizationFields } from '../generated/contentfulTypes'
 
 export type LogoUrl = string | null
 
-export type OrganizationLogoDataLoader = DataLoader<string, LogoUrl, string>
+interface OrganizationLogoLoadInput {
+  field: keyof IOrganizationFields
+  key: string
```

Comment:
I find this field and key prop names confusing. Essentially the field denotes the key and the key denotes the value.

How about renaming this to be
field: keyof IOrganizationFields
value: string

Reply:
yes, makes sense 🤝